tests/tracing/test_misc.py

```diff
@@ -246,3 +246,18 @@ def test_has_tracestate_enabled(sentry_init, tracestate_enabled):
         assert has_tracestate_enabled() is True
     else:
         assert has_tracestate_enabled() is False
+
+
+def test_set_meaurement(sentry_init, capture_events):
+    sentry_init(traces_sample_rate=1.0, _experiments={"custom_measurements": True})
+
+    events = capture_events()
+
+    transaction = start_transaction(name="measuring stuff")
```

Comment:
if I do:

`child = transaction.start_child(x)`

can I do `child.set_measurement(x, y)` too?

Or `start_child` would return a different interface that does not provide `set_measurement`? since measurements are only available at the transaction level.

Reply:
Currently they are just on transaction and not span, `start_child` returns a span so you can't add measurements there. From what I understood of the requirements, we only want to expose this api on the transaction but this is a design decision and we can also change it.

Open decision is also if we want to expose this on the hub/top-level api as well where it'll just fetch the current hub/scope/transaction and set it there.

Reply:
I agree that we should not leak this API to the Child level as well, but a few SDKs return the very same interface for `start_transaction` and `start_child`, so unless we change that, the public API will leak.
cc @bruno-garcia